src/services/email/ses-email-templates/index.js

```diff
@@ -358,11 +371,11 @@ Summary:
 or your response to a SPA Request for Additional Information (RAI)). You can 
 expect a formal response to your submittal to be issued within 90 days, 
 before {{ninetyDaysDate}}.</p>
-<p>This mailbox is for the submittal of State Plan Amendments and non-web 
-based responses to Requests for Additional Information (RAI) on submitted 
-SPAs only. Any other correspondence will be disregarded.</p>
+<p>This mailbox is for the submittal of Section 1915(b) and 1915(c) Waivers, 
```

Comment:
Need to double check this, the section seems to be for a Medicaid SPA RAI response so I dont think the text for waivers would be appropriate.

Reply:
You are right, I double checked with the confluence and changed it to match. Thanks for catching that :)